### PR TITLE
feat: safe implementation

### PR DESCRIPTION
**Pull request by chris13524**
- [ ] Get new test passing

### CODE REVIEW

File: crates/yttrium/src/transaction/send/simple_account_test.rs [hunk 1]
```diff
@@ -238,23 +238,26 @@ mod tests {
...
+        println!("Received User Operation receipt: {:?}", receipt);
 
-        // println!("Querying for receipts...");
+        println!("Querying for receipts...");
 
-        // let receipt = bundler_client
-        //     .wait_for_user_operation_receipt(user_operation_hash.clone())
-        //     .await?;
+        let receipt = bundler_client
+            .wait_for_user_operation_receipt(user_operation_hash.clone())
```
Comment (jackpooleywc) [inline crates/yttrium/src/transaction/send/simple_account_test.rs:253]: Last time I tried to run this, I was getting an error, which is why it was commented out, is this now working?

File: test/scripts/forked_state/docker-compose.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ services:
     image: ghcr.io/foundry-rs/foundry:nightly-f6208d8db68f9acbe4ff8cd76958309efb61ea0b
     restart: unless-stopped
     ports: ["8545:8545"]
-    entrypoint: [ "anvil", "--fork-url", "https://gateway.tenderly.co/public/sepolia", "--host", "0.0.0.0", "--block-time", "0.1", "--gas-price", "1" ]
+    entrypoint: [ "anvil", "--fork-url", "https://gateway.tenderly.co/public/sepolia", "--host", "0.0.0.0" ]
```
[the remark this answers is not in the document]
Comment (chris13524) [reply test/scripts/forked_state/docker-compose.yaml:6]: Fast block time was causing a lot of CPU usage on my machine. Don't think we need a custom gas price

File: crates/yttrium/src/bundler/models/user_operation_receipt.rs [hunk 3]
```diff
@@ -15,7 +15,7 @@ pub struct UserOperationReceiptReceipt {
     pub contract_address: Option<String>,
     pub status: String,
     pub logs_bloom: String,
-    pub r#type: String,
+    // pub r#type: String,
```
[the remark this answers is not in the document]
Comment (chris13524) [reply crates/yttrium/src/bundler/models/user_operation_receipt.rs:18]: The local bundler response didn't include it, and we don't need it

File: crates/yttrium/src/transaction/send/simple_account_test.rs (hunk 1 above)
Comment (chris13524) [reply crates/yttrium/src/transaction/send/simple_account_test.rs:253]: Yeah it's working now

**Review by jackpooleywc — APPROVED** [review]
:shipit:

**Review by jackpooleywc — APPROVED** [review]

**Pull request merged**